Dockerfile.jupyter

```diff
@@ -0,0 +1,17 @@
+FROM jupyter/base-notebook:d0b2d159cc6c
+
+USER $NB_USER
+
+RUN pip install sparkmagic
+
+RUN mkdir /home/$NB_USER/.sparkmagic
+COPY sparkmagic/example_config.json /home/$NB_USER/.sparkmagic/config.json
+RUN sed -i 's/localhost/spark/g' /home/$NB_USER/.sparkmagic/config.json
+RUN jupyter nbextension enable --py --sys-prefix widgetsnbextension
+RUN jupyter-kernelspec install --user $(pip show sparkmagic | grep Location | cut -d" " -f2)/sparkmagic/kernels/pyspark3kernel
```

Comment:
Fixed, by setting up R properly. Tested and works!

![2017-05-26-11 42 44](https://cloud.githubusercontent.com/assets/14872/26501658/362d0adc-4208-11e7-8e53-7e4c6eec6dab.png)